draft-pauly-ohai-svcb-config.md

```diff
@@ -249,8 +249,15 @@ key configurations per-client MUST NOT be used.
 
 When clients fetch a gateway's configuration using the well-known URI,
 they can expose their identity in the form of an IP address if they do not
-connect via a proxy or some other IP-hiding mechanism. Clients SHOULD
-use a proxy or similar mechanism to avoid exposing client IPs to a gateway.
+connect via a proxy or some other IP-hiding mechanism. In some circumstances,
+this might not be a privacy concern, since revealing that a particular
+client IP address is preparing to use an Oblivious HTTP service can be
+expected. However, if a client is otherwise trying to obfuscate its IP
+address or location (and not merely decouple its specific requests from its
+IP address), or revealing its IP address will increase the risk of a key
+targeting attack (if a gateway service is trying to differentiate traffic
+across client IP addresses), a proxy or similar mechanism can be used to fetch
+the gateway's configuration.
```

Comment:
In a typical DNR/DDR use case, the oblivious target will be deployed by the network-provider and it not only knows the client IP address but also the client identity (e.g., Network Access Identifier (NAI) or user identifier). If the client fetches the key configuration from the oblivious target directly, it would know the client identity interested to use the oblivious target and can be subjected targeted attacks (e.g., key targeting attack for an specific user in a mobile network).